Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.19.9-alpine as builder
+WORKDIR /src
+COPY . . 
+RUN cd cmd/apmsoak \
+  && go mod download \
+  && go build
```

Comment:
We might benefit from having this defined in Makefile in the future. Just sharing the reference [here](https://github.com/elastic/apm-server/blob/main/testing/apmsoak/Makefile#LL20C2-L20C2)

Reply:
That's true